### ISSUE TITLE
Progress loading bar lacks of accessibility

### ISSUE THREAD

**Issue by btzr-io** [btzr-io/godot-aria#3]
User reports form discord:

> It seemd to work, though the progress bar reads in scientific notation sometimes. NVDA might do that automatically when numbers reach a certain amount of precision, dunno. I'd recommend a percentage.

> My results show that only Firefox is saying percentages for the progress bar, while the Chromium based browsers are using scientific notation. Either way though the numbers get pretty wild since they aren't being rounded at first.

> Honestly? Yeah it kind of is annoying. Which I hate to say because I know you worked on that to make it better. But it's not really necessary unless you're on dialup, and just ends up spamming the TTS.

> Ah, gotcha. Yeah progress bars are nice, but I only like them for really big installs or downloads my self. A loading message is totally fine for something like this.
